elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -42,6 +42,19 @@ $(function() {
                             <label for="modal-{{uuid}}-themes">Editor Theme</label>
                             <select class="custom-select" id="modal-{{uuid}}-themes">
                             </select>
+
+                            <div id="modal-split" style="padding-top: 1rem"></div>
+                            <label for="modal-{{uuid}}-font-adjust">Font Size</label>
+                            <br>
+                            <button type="button" class="btn btn-sm btn-secondary textsize-dec" aria-label="fontsize decrease">
+                                <i class="fa fa-font"></i>
+                                -
+                            </button>
+                            <span class="textsize-current" id="modal-{{uuid}}-textsize" style="padding: 3px;"></span>
+                            <button type="button" class="btn btn-sm btn-secondary textsize-inc" aria-label="fontsize increase">
+                                <i class="fa fa-font"></i>
+                                +
+                            </button>
```

Comment:
The increase/decrease buttons are nice, but I think it may be easier to make this accessible if we use a `<select>` element with predefined text sizes. I'm thinking maybe the following (based on https://github.com/ajaxorg/ace/blob/23208f2f19020d1f69b90bc3b02460bda8422072/src/ext/textarea.js#L390, with a larger max value):

- 10px
- 11px
- 12px
- 14px
- 16px
- 18px
- 20px
- 22px
- 24px